.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish prefect-saturn to PyPI
+on:
+  release:
+    types:
+      - published
+jobs:
+  build-and-publish:
+    name: Build and publish prefect-saturn to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install build dependencies
+      run: >-
+        python -m
+        pip install
+        setuptools
+        wheel
+        --upgrade
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python
+        setup.py
+        sdist
+        bdist_wheel
+    - name: Publish distribution to PyPI
+      if: startsWith(github.event.ref, 'refs/tags')
```

Comment:
You probably don't need this if you are only doing the action on publish.

Reply:
ooooo right!

Reply:
I just removed this in https://github.com/saturncloud/prefect-saturn/pull/7/commits/16352452df6b6a083b9971bea5e144c024105711